### DISCUSSION TITLE
Presentation of visualizations (titles, dropdowns)

### DISCUSSION THREAD

**Discussion by BeritJanssen** [UUDigitalHumanitieslab/I-analyzer#1425] [General]
This is really something to discuss first, but I wonder if we couldn't adjust the visualization titles as follows:
- Document and term frequency visualizations: Right now, they say `frequency of '{query}' by {Field} (n of documents/terms)`. This is repeating information found elsewhere on screen. The field name is on the x axis, and the query is in the search field. I would suggest to make the title `Document frequency` and `Term frequency`, respectively, and also call it by these terms in the visualization dropdown.
- Ngram visualization: It's now called "neighbouring words", we refer to it internally often as ngrams. Would it be fair or misleading to use the term "collocations? Whatever we decide, it would be good to add this title to the ngram graph as well (right now, we have only axis labels in there)
- Instead of the visualization menu item "most frequent words", I am in favour of going back to "wordcloud", as it makes it very explicit to the user what to expect. The title (whichever we choose) should also be added to the graph.

My argument would be that the menu items and titles should correspond, and that that they should be short, while of course they also shouldn't be misleading.

**Comment by lukavdplas** [comment]
Some thoughts:

- The term frequency graph is called 'frequency of the search term' in the graph rather than 'term frequency' because 'term frequency' is also a fitting description of the word cloud (i.e. the frequency of all terms). This was intended to disambiguate the two.
- I think it would be misleading to say 'collocations' for the ngram graph, since it does not show collocations. What is shows are _ngrams that contain the search term_. That could be added to the title/caption though.
- Using 'most frequent words' rather than 'wordcloud' was an attempt to describe the information the user might be after, rather than the form in which it is presented. I still think this is the better option, but I can see how 'wordcloud' may be more evocative to some users who have already seen the graph. (As a side note, using a visual descriptor like 'word cloud' is somewhat of a no-no re. accessibility for screen reader users, but not a huge deal.)
- I agree that the titles tend to repeat information that is found elsewhere. The intention for having titles in the first place this was downloads, but we are not really offering the level of customisation to make paper-ready graphs, so the png download is not really such a core feature. Perhaps we could make a caption with a show/hide option. So it's hidden by default, but for screenshots/downloads where you won't have the context of the rest of the page, you can show the caption and include it.

**Comment by BeritJanssen** [comment]
Perhaps we could also get some insights if we hold a questionnaire among users (see https://github.com/UUDigitalHumanitieslab/I-analyzer/discussions/1415)
